docs/source/sci_api_tutorial/armer.rst

```diff
@@ -2,21 +2,262 @@
+    
+``env_settings``
+     env_settings states environment settings of external software (e.g., env_settings = ‘’’module load Gaussian/16.B.01’’’).
+    
+    .. admonition:: How to obtain
+        
+        Same as `commands`, different subtasks require different environments. 
+     
+
+Output
+------------------------------------------------
+A job script will be generated by the “workflow script” using commands implemented in ARMer.  
+
+
+Arguments
+==============================================
+
+.. dropdown:: :fa:`eye,mr-1` Click to see full argument explanations
+    
+    1. For Job Configuration:
+
+    ``commands``
+        refers to the target shell commands for running external software for a specific enzyme modeling sub-task
+
+    ``cluster``
+        refers to an HPC class object that contains miscellaneous details about user’s local HPC
+        
+    ``env_settings``
+        states environment settings of external software 
+
+    ``res_keywords``
+        configures computing resources for the job (including parameters such as `core_type`, `nodes`,`nodes_core`,etc), which can be referenced from `Input/Output <#input-output>`_ section.
+    
+    ``jobs``
+        a list of ClusterJob object to be execute
+
+    ``period``
+        the time cycle for detect job state (Unit: second)
+        
+    
+    2. For Job Submission:
+
+    ``sub_dir`` 
+        dir for submission. commands in the sub script usually run under this dir.
+                
+    ``script_path`` 
+        path for submission script generation.
+        (default: sub_dir/submit.cmd; will be sub_dir/submit_#.cmd if the file exists. # is a growing index)
+                
+    
+    3. For Job Monitoring:
+    
+    There are two functions: "wait_to_end" (single submission) and "wait_to_array_end" (array submission)
+        
+    ``period``
+        the time cycle for update job state change (Unit: s)
+    
+    The following arguments are array submission only:
+
+    ``jobs``
+        a list of ClusterJob object to be execute
+        
+    ``array_size``
+        
+        how many jobs are allowed to submit simultaneously. 
+
+        (e.g. 5 for 100 jobs means run 20 groups. All groups will be submitted and 
+        in each group, submit the next job only after the previous one finishes.)
+        
+    ``sub_dir``
+        (default: self.sub_dir)
+
+        submission directory for all jobs in the array. 
+        
+        Overwrite existing self.sub_dir in the job obj
+        
+        * you can set the self value during config_job to make each job different
+    
+    ``sub_scirpt_path`` 
+        (default: self.sub_script_path)
+        
+        path of the submission script. Overwrite existing self.sub_script_path in the job obj
+        
+        * you can set the self value during config_job to make each job different
+        
+
+Example Code
```

Comment:
Use examples from https://enzyhtp-doc.readthedocs.io/en/latest/sci_api_tutorial/single_point.html
But here we focus on the `cluster_job_config `